lib/elasticity/version.rb

```diff
@@ -1,3 +1,3 @@
 module Elasticity
-  VERSION = '2.8.0'
+  VERSION = '2.9.0'
```

Comment:
I think we've already diverged from upstream a lot, no? https://github.com/sewichi/elasticity/commits/master/lib/elasticity/version.rb